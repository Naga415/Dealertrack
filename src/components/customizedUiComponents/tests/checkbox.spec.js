import React from "react";
import renderer from "react-test-renderer";
import Checkbox from "../checkbox";

const onChange = jest.fn();

describe("Checkbox", () => {
  it("renders <Checkbox />", () => {
    const props = {
      htmlId: "Checkbox",
      name: "Checbox Name",
      label: "Dummy Label",
      checked: false,
      disabled: false,
      onChange: () => { onChange;},
      actions: {}
    };

    const component = renderer.create(<Checkbox {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
