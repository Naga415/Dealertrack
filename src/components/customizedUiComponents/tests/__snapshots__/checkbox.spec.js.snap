// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders <Checkbox /> 1`] = `
<div
  className="checkbox"
>
  <label
    htmlFor="Checbox Name"
  >
    <input
      checked={false}
      disabled={false}
      name="Checbox Name"
      onChange={[Function]}
      type="checkbox"
    />
    Dummy Label
  </label>
</div>
`;
